server/app/main.py

```
@@ -0,0 +1,57 @@
+import click
+import structlog
+import uvicorn
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+
+from . import settings
```

Pylint (3.3.7)

[error] No name 'settings' in module ''

(E0611)